tools/SetupFlow/DevHome.SetupFlow/ViewModels/QuickstartPlaygroundViewModel.cs

```diff
@@ -604,6 +610,17 @@ public void OnQuickstartSelectionChanged()
             ConfigureForProviderSelection();
         }
     }
+
+    [RelayCommand]
+    private void OnLoaded()
+    {
+        var isEnabled = _experimentationService.ExperimentalFeatures.FirstOrDefault(f => string.Equals(f.Id, "QuickstartPlayground", StringComparison.Ordinal))!.IsEnabled;
+        if (!isEnabled)
+        {
+            var setupFlow = Application.Current.GetService<SetupFlowViewModel>();
+            setupFlow.ResetToMainPage();

```

Comment:
I was testing out the changes locally since I was curious to see what the 'flash' experience was like when navigating back to the Machine configuration page. 

At least on my device, if it was happening, it was faster than my eyes could detect! (This was on a cloud desktop, so over RDP.) Clicking on the Machine configuration page showed the default landing page as expected. 

I did however see some binding errors getting generated, which I couldn't immediately parse. This was accompanied by a bit of UI unresponsiveness. I'll try to grab a video to show what I'm seeing. 

Count | Data ContextBinding Path | TargetTarget Type | Description
-- | -- | -- | --
22 | Windows.Foundation.IReference 1 Breadcrumbs | Microsoft.Ul.Xaml.Controls.BreadcrumbBar.ItemsSource, Name='BreadcrumbBar' Object | BindingExpression path error: 'Breadcrumbs' property not found on'Windows.Foundation.IReference'1'

Reply:
Ah, interesting. I'm not seeing the exact error, but my error settings are probably different. I'm curious to see the video

Reply:
Nit: Curious if that one frame will still show up if we use the `Loading` event instead of `Loaded`.

Reply:
Here's the video. There is some other UI unresponsiveness which got picked up here from the 6 second to the 24 second(!) mark where I couldn't navigate to the Settings menu; this was markedly slower than when I had originally tried. The more salient part is around 30 seconds in where I navigate back to machine configuration, and then try to go to the dashboard (this is what I was originally referring to in my prior comment; you can see how long it is taking for the banner image to load). You can see the binding errors accumulating in the application toolbar (I ran out of time just now, but I'd like to dig a bit more to see if I can tease apart what views those errors are associated with, as well as if they repro on main). 

https://github.com/user-attachments/assets/e6d262b1-d952-4c71-926b-87707ec0ea7e

Reply:
For the loaded/loading substitution, loading throws an exception when I navigate to the QSP page after the feature has been disabled. After syncing offline with, it seems like there is a possibility of a chance of exception that things might not have yet loaded. So keeping with loaded for now.